cs.json

```diff
@@ -0,0 +1,347 @@
+                "reminders" : " připomínky",
+                "history" : " historie obchodu",
+                "upcoming" : " nadcházející",
+                "sets" : " sady předmětů"
+            },
+            "guideDropdown" : "návod",
+            "news" : " novinky ve hře",
+            "randomiser" : " randomizér",
+            "about" : {
+                "dropdown" : "o nás",
+                "translate" : " překlad",
+                "about" : " o stránce fnbr.co",
+                "privacy" : " zásady ochrany osobních údajů",
+                "donate" : " Přispět",
+                "discord" : "Discord",
+                "twitter" : "Twitter"
+            },
+            "api" : "api",
+            "account" : {
+                "dropdown" : "účet",
+                "account" : "účet",
+                "logout" : " odhlásit se"
+            },
+            "languagePicker" : {
+                "trigger" : " jazyk",
+                "title" : " Výběr jazyka",
+                "currentLanguage" : " Aktuálně používáte: [language]",
+                "error" : " Chyba při stahování souboru s jazykem",
+                "authors" : " Přeložil:",
+                "cta" : " Kliknutím přepnete na: [language]",
+                "switching" : " Stahování [language]...",
+                "finished" : " Váš jazyk byl změněn!"
+            }
+        },
+        "home" : {
+            "title-welcome" : " Vítejte na stránkách",
+            "line1-welcome" : " fnbr.co je neoficiální seznam s kosmetiky pro Fortnite: Battle Royale.",
+            "line2-welcome" : " Prohlédnout [aktuální item shop](/shop), a seznam [veškerá dostupné kosmetiky](/cosmetics), a mnoho dalšího.",
+            "title-countdown" : "Resetovat Item Shop:",
+            "link-countdown" : " Prohlédnout Item Shop&rarr;",
+            "title-upcoming" : " Nevydané itemy",
+            "line1-upcoming" : " Do souborů hry byly nedávno přidány tyto itemy.",
+            "line2-upcoming" : " Ačkoli ještě nejsou nutně zveřejněny, měly by se objevit v některé z budoucích aktualizací.",
+            "error-upcoming" : " V tuto chvíli nelze načíst nevydané itemy<br />Zkuste to prosím za chvíli znovu. Děkujeme!",
+            "button-upcoming" : " Zobrazit všechny nevydané Itemy &rarr;",
+            "title-trending" : " Trendy Itemy ",
+            "line1-trending" : " Seznam nejnovějších očekávaných položek podle připomínek přidaných v naší aplikaci.",
+            "error-trending" : " Momentálně nelze načíst trendy itemy<br />Zkuste to prosím za chvíli znovu. Děkujeme!",",
+            "button-trending" : " Zobrazit další trendy itemy &rarr;",
+            "title-all" : " Všechny Itemy",
+            "line1-all" : " Máme seznam všech itemů ze hry Fortnite: Battle Royale. Můžete je najít na stránce: [zde](/list).",
+            "button-all" : " Zobrazit všechny itemy &rarr;",
+            "title-news" : " Nedávné [fortnitenews.com](https://fortnitenews.com|newtab=true) articles",
```

Comment:
Is `articles` really the native translation (aka same word)?

Reply:
I misspelled the word: the **articles** are in our language: **články**